css/card.css

```diff
@@ -0,0 +1,189 @@
+.card-flex {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.card-shadow:hover {
+  box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.2), 15px 17px 60px var(--grey-bg);
+  transition-delay: 100ms;
+}
+
+.card {
+  border-radius: var(--size-1);
+  background-color: var(--card-bg);
+  border: 2px solid var(--grey-bg);
+  margin: var(--size-1) var(--size-1);
+}
+
+.discount {
+  color: var(--disc-green);
+}
+
+.card-text-only h2 {
+  margin: 0 0 1rem 0;
+}
+
+.card-text-only p {
+  margin: 0.7rem 0;
+}
+
+.card-text-only {
+  display: block;
+  padding: var(--size-1);
+  height: 25rem;
+  width: 30rem;
+  height: fit-content;
+  width: fit-content;
+  max-width: 35rem;
+  min-height: 10rem;
+}
+
+
+.card-hz-btn .button {
+  width: 14rem;
+  margin-left: 0;
+}
+
+.card-hz {
+  max-width: 35rem;
+}
+
+.hz-card-img {
+  max-width: 25rem;
+  max-height: 20rem;
+}
+
+.hz-card-text {
+  width: 50%;
+}
+
+.card .button:hover {
+  box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.2);
+}
+
+.vt-card-img {
+  max-width: 20rem;
+  max-height: 30rem;
+}
+
+.card-vt-btn {
+  justify-content: space-between;
+}
+
+.card-vt-btn .button {
+  width: 10rem;
+  margin: 0;
+  font-size: 0.75rem;
+}
+
+.card-vertical {
+  width: 23rem;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.rating {
+  text-align: center;
+}
+
+.ecomm-card .vt-card-img {
+  max-width: 10rem;
+  max-height: 13rem;
+  width: 10rem;
+  height: 13rem;
+  object-fit: cover;
+}
+
+.card.ecomm-card {
+  max-width: 12rem;
+}
+
+.card.ecomm-card .vt-card-text {
+  text-align: left;
+}
+
+.ecomm-card .card-price .disc-price {
+  font-size: 0.7rem;
+  color: var(--grey-border);
+}
+
+.ecomm-card-heading {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.9rem;
+  font-weight: 500;
+}
+
+.img-badge-container {
+  position: relative;
+}
+
+.vt-card-badge {
+  background-color: var(--success-color);
+  border-radius: 0.3rem;
+  font-weight: 500;
+  position: absolute;
+  top: var(--size-1-5);
+  left: var(--size-1);
+}
+
+.ecomm-card .vt-card-badge {
+  background-color: var(--success-color);
+  border-radius: 0.3rem;
+  font-weight: 500;
+  position: absolute;
+  top: 5px;
+  left: 0.1px;
+}
+
+.card .btn-close {
+  position: absolute;
+  background-color: var(--white-bg);
```

Comment:
Instead of giving the color name to a variable, you can use `light-bg` or `bg-color-1,bg-color2.`